src/index.js

```diff
@@ -49,8 +50,9 @@ const startServer = async () => {
     const romName = rom.replace(/\.[a-zA-Z]+$/, '')
 
     twitchShufflerListener.say(`/me Swapping to "${romName}" (${cause})`);
+    message = `switchRom\t${rom}\n`;
     sockets.forEach((sock) => {
-      sock.write(`switchRom\t${rom}\n`);
+      sock.write((message.length - 1) + ' ' + message);
```

Comment:
Curious about this change as well. I don't see a change on the command protocol. Is this required for the updated changes to work?

Reply:
This is required for BizHawk 2.6.2+. I found a note in the comm.socketServerResponse function describing the change.

Reply:
![Screenshot_2024-02-10_16-00-31](https://github.com/alexjpaz-twitch/bizhawk-crowd-shuffler/assets/6464463/8b1d84db-7c59-4f7a-8e33-f653ea6cd0b3)

Reply:
Cool! Thanks for digging into it.